.github/workflows/update-integration-tests.yml

```diff
@@ -1,11 +1,6 @@
 name: Update Playwright Snapshots
 
 on:
-  workflow_dispatch:
-    inputs:
-      number:
-        description: 'PR number'
-        required: true
```

Comment:
You may want to keep this, but the body of action would probably need some more changes. I recommend merging as-is and restoring the `workflow_dispatch` later.